src/compressed_tensors/compressors/model_compressor.py

```diff
@@ -200,6 +201,7 @@ def parse_quantization_config(compression_config: Dict) -> Union[Dict, None]:
         # SparseAutoModel format
         quantization_config = deepcopy(compression_config)
         quantization_config.pop(SPARSITY_CONFIG_NAME, None)
+        quantization_config.pop(COMPRESSION_VERSION_NAME, None)
```

Comment:
Eventually we will want to use this value to check for breaking changes/minimum support, but fine for now

Reply:
Yep, all these lines are saying is "everything that is not the sparsity config and version is the quantization config"